### ISSUE TITLE
How to improve the accuracy of transcribing a telephony audio?

### ISSUE THREAD

**Issue by ML6634** [facebookresearch/wav2letter#599]
May I ask which wav2letter model might be better for transcribing a telephony audio? Anyway for us to use telephony data to train the model? Or, any other possible ways to improve the accuracy of transcribing a telephony audio? For a project like this, may I use Python instead of Ubuntu commands? Thank you!

**Comment by ML6634** [comment]
@avidov @tlikhomanenko

Would it be possible for me to train the model using my own telephony wav file? Thank you in advance!

**Comment by tlikhomanenko** [comment]
Hi @ML6634,

You can use any pre-trained model we published to see what the quality will be: the best to try is a model trained on Librispeech + LibriLight data from here https://github.com/facebookresearch/wav2letter/tree/master/recipes/models/sota/2019, use at start https://dl.fbaipublicfiles.com/wav2letter/sota/2019/librivox/models/am/am_transformer_ctc_librivox_dev_other.bin. But still transferring to out-of-domain data could be not ideal, so maybe you will need to finetune this model on your own data (make sure to use correct sample rate which is 16kHz). 

Another solution is to train one of the architectures (like transformer + CTC) from https://github.com/facebookresearch/wav2letter/tree/master/recipes/models/sota/2019 on your own data, don't see any problem to do so. 

I would try at first eval our pre-trained models and try to finetune on your data. For more details on results of transfer learning, please have a look at https://github.com/facebookresearch/wav2letter/issues/577.

If you need model for the inference pipeline at the end, @avidov can provide you more details on this and also check the above issue.

About python: we don't have python for the training pipeline, so you need to run command line. How do you want to use python (example of operations/steps)?

**Comment by ML6634** [comment]
At ~/wav2letter/build$, how to install Train?

Regarding Python, I would say Ubuntu command line would be great for now.  Thank you!

**Comment by tlikhomanenko** [comment]
Inside build folder you will have binary files: Train, Test and Decoder. To run training you just run 
```
~/wav2letter/build/Train [parameters]
```

Example here https://github.com/facebookresearch/wav2letter/wiki/Train-a-model, or for some particular model https://github.com/facebookresearch/wav2letter/blob/master/recipes/models/sota/2019/librispeech/README.md

**Comment by ML6634** [comment]
In my build folder, I do not have Train, Test, and Decoder. How to make these three binary files?

**Comment by tlikhomanenko** [comment]
You need to run cmake and make commands. Details what you need to install as dependencies https://github.com/facebookresearch/wav2letter/wiki/Dependencies 
and then how to build w2l https://github.com/facebookresearch/wav2letter/wiki/General-building-instructions. 

Another alternative is to use docker image (where everything is built for you) https://github.com/facebookresearch/wav2letter/wiki/Building-Running-with-Docker

**Comment by tlikhomanenko** [comment]
Feel free to reopen if further assistance is needed.
